.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report 🐞
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+[comment]: <> (If the issue happens in multiple SDK, open it in the integration-guide instead: https://github.com/meilisearch/integration-guides/issues)
```

Comment:
```suggestion
[comment]: <> (If the issue happens in multiple SDKs, open it in the [integration-guide](https://github.com/meilisearch/integration-guides/issues) instead: )
```

Reply:
It's a comment so the markdown won't be working. I commit the suggestion anyway it's felt cleaner but we can also take off the comment [comment] so the text would appear.

Reply:
We should not let it as it is if it does not work :) as you wish: either you go back to your previous suggestion, or you remove the `comment` :)

Reply:
Just noticed it: `integration-guide` -> `integration-guides`

Reply:
Done